NetworkCloud documents update - az cli samples update (#108100)
Azure/azure-content: articles/operator-nexus/howto-cluster-metrics-configuration-management.md

## How to manage cluster metrics configuration

To support the lifecycle of cluster metrics configurations, the following interactions allow for the creation and management of a cluster's metrics configurations.

> ### Creating a metrics configuration using `az networkcloud`

Reviewer: ```suggestion
### Creating a metrics configuration
```
please add the definitions for each of the parameters used below. you can use the other pages for the reference (i think a table is used?).

Use the `az network cluster metricsconfiguration create` command to create metrics configuration for cluster. If you have multiple Azure subscriptions, select the appropriate subscription ID using the [az account set](/cli/azure/account#az-account-set) command.

```azurecli
az networkcloud cluster metricsconfiguration create \
 --cluster-name "<CLUSTER>" \
 --extended-location name="<CLUSTER_EXTENDED_LOCATION_ID>" type="CustomLocation" \

>  --location "location" --collection-interval 15 \

Reviewer: ```suggestion
 --location "<LOCATION>" \
 --collection-interval <COLLECTION_INTERVAL (1-1440)> \
```

 --enabled-metrics "<METRIC_TO_ENABLE_1>" "<METRIC_TO_ENABLE_2>" \

>  --tags <TAG_KEY1="<TAG_VALUE1" <TAG_KEY2>="<TAG_VALUE2>" \

Reviewer: ```suggestion
 --tags <TAG_KEY1="<TAG_VALUE1>" <TAG_KEY2>="<TAG_VALUE2>" \
```
although please check if tags need to be mentioned here. seems nonessential.

 --resource-group "<RESOURCE_GROUP>"
```

> ### Creating a metrics configuration using `az rest`

Reviewer: i don't think we need to keep az rest in here. this was a temporary workaround while waiting for CLI

Specifying `--debug` in REST API will result in the tracking operation status in the returned command output. This operation status can be queried to monitor the progress of the operation. See: [How-to track asynchronous operations](howto-track-async-operations-cli.md).

> [!NOTE] 
> * The default metrics collection interval for standard set of metrics is set to every 5 minutes. Changing the `collectionInterval` will also impact the collection frequency for default standard metrics.

> > * The only name allowed for the metricsConfiguration is `default`.

Reviewer: ```suggestion
> * There can be only one set of metrics configuration defined per cluster. The resource is created with the name `default`.
```

## Retrieving a metrics configuration

> ### Retrieving a metrics configuration using `az networkcloud`

Reviewer: ```suggestion
### Retrieving a metrics configuration
```

```azurecli
az networkcloud cluster metricsconfiguration show \
 --cluster-name "<CLUSTER>" \
 --resource-group "<RESOURCE_GROUP>"
```

> ### Retrieving a metrics configuration using `az rest`

Reviewer: remove

## Updating a metrics configuration

> ### Updating metrics configuration using `az networkcloud`

Reviewer: ```suggestion
### Updating metrics configuration
```

> This command is used to patch properties of the provided metrics configuration of cluster, or update the tags assigned to the metrics configuration.Properties and tag updates can be done independently.

Reviewer: ```suggestion
This command is used to patch properties of the provided metrics configuration of cluster, or update the tags assigned to the metrics configuration.
```
again not sure we need to mention tags here and in the command

```azurecli
az networkcloud cluster metricsconfiguration update \
 --cluster-name "<CLUSTER>" \

>  --collection-interval 15 \

Reviewer: ```suggestion
 --collection-interval <COLLECTION_INTERVAL (1-1440)> \
```

 --enabled-metrics "<METRIC_TO_ENABLE_1>" "<METRIC_TO_ENABLE_2>" \
 --tags <TAG_KEY1>=<TAG_VALUE1 <TAG_KEY2>=><TAG_VALUE2> \
 --resource-group "<RESOURCE_GROUP>"
```

> ### Updating metrics configuration using `az rest`

Reviewer: remove

## Deleting a metrics configuration

> ### Deleting metrics configuration using `az networkcloud`

Reviewer: ```suggestion
### Deleting metrics configuration
```

```azurecli
az networkcloud cluster metricsconfiguration delete \
 --cluster-name "<CLUSTER>" \
 --resource-group "<RESOURCE_GROUP>"
```

> ### Deleting metrics configuration using `az rest`

Reviewer: remove

```

Specifying `--debug` in REST API will result in the tracking operation status in the returned command output. This operation status can be queried to monitor the progress of the operation. See: [How-to track asynchronous operations](howto-track-async-operations-cli.md).

> ### Parameters for cluster metrics configuration operations

Reviewer: add <COLLECTION_INTERVAL (1-1440)>
i think it is better to put these within create block. it would read better

| Parameter name                        | Description                                                                                                           |
| --------------------------------------| --------------------------------------------------------------------------------------------------------------------- |

> | CLUSTER                               | Resource Name of the Cluster                                                                                          |

Reviewer: go over all descriptions and clean them up:
- Create should be lower cased.
- Cluster should be capitalized everywhere
